language/move-lang/tests/move_check/expansion/standalone_mname_with_type_args.exp

```diff
@@ -3,10 +3,10 @@ error:
    ┌── tests/move_check/expansion/standalone_mname_with_type_args.move:3:22 ───
    │
  3 │         let _ = 0 + M<u64>;
-   │                           ^ Unexpected token: ';'
+   │                           ^ Unexpected ';'
    ·
  3 │         let _ = 0 + M<u64>;
-   │                           - Expected: '{', '('
+   │                           - Expected either a brace-enclosed pack expression or a parenthesized list of arguments for a function call
```

Comment:
I don't usually use the term 'pack' anywhere. I usually use the term 'struct instantiation'  or 'construction'

Reply:
But its tricky because this could appear on the LHS and the parser wouldn't know. i.e we should have a test for 
```
let M<u64> = foo();
```
And the error message is going to have to work in both cases

Reply:
The test we've already got (invalid_unpack_assign_rhs_not_fields.move) happens on the RHS of an assignment, and I just didn't notice that the message doesn't work well in that context. (I was going off the code which creates an Exp_::Pack.) I'll replace "pack expression" with "list of field expressions". I'm not confident that "field expression" is totally clear, but this code is already using that term for a different error message. Whenever we document the syntax, we'll have to figure out what terminology to use. If you've already got a better option, I'll be happy to adopt it.

Reply:
'list of field expressions' or 'fields expression' or 'field list expression' all seem ok.

But agreed we will need to fix up a lot of this terminology consistency later